File: LeetCode/Arrays/Easy/2011. Final Value of Variable After Performing Operations/main.py
class Solution:
    def finalValueAfterOperations(self, operations):
        ans = 0
        for x in operations:
            if x == "++X" or x == "X++":
                ans += 1
            else:
                ans -= 1
        return ans


s = Solution()
print(s.finalValueAfterOperations(["--X", "X++", "X++"]))

File: LeetCode/Arrays/Easy/2011. Final Value of Variable After Performing Operations/main.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution
{
public:
    int finalValueAfterOperations(vector<string> &operations)
    {
        int ans = 0;
        for (int i = 0; i < operations.size(); i++)
        {
            (operations[i][1] == '+') ? ans++ : ans--;
        }
        return ans;
    }
};

int main()
{
    vector<string> input{"--X", "X++", "X++"};
    Solution obj;
    int ans = obj.finalValueAfterOperations(input);
    cout << ans;
}